test-data/unit/check-qualname.test

```diff
@@ -0,0 +1,10 @@
+[case testQualNameTypeCheck]
+
+# flags: --python-version 2.7
+class C:
+    NAME = __qualname__ # E: NameError: name '__qualname__' is not defined
+
+# flags: --python-version 3.3
```

Comment:
You can't have two separate frag sets in the same test case, you need to have a separate one.